### ISSUE TITLE
Open source Complice?

### ISSUE THREAD

**Issue by kbex1** [aspiers/complice#21]
Read through your "legal stuff" and there is a sentiment there that states that "Complice is different" in that, that Complice goal is aligned with it's user base and "it's not out to get you". which is a fine sentiment at the moment with this current leadership but nothing can be said to future generations of Complice leadership. right now the users only option is to trust you, the company, that it will stay that way.
trust which has been broken many times before by many platforms (including this one). we know where we end up when we trust propriety closed companies.

The only way (as far as i can see) when it comes to eliminated that trust issue and make sure the company stays aligned with it's user base is to open source the project. if something down the line changes and gets misaligned with it's user base we can fork it and keep the project open and truly aligned with it's user base.

i would be happy to pay your monthly fee if i knew that it was a free/open project.  you have a unique solution which you clearly put a lot of work into. it's very nuanced and i appreciate seeing things like that online.
The only reason i'm writing this is because Complice is stating it's different. If it is, then it should have no fear in opening the project up.
users (at least ones that think about their information and data) will be happy to engage with companies that hold the same notions and ideals.

Thank you for reading this.

p.s 
consider using Sourcehut over github in the future.

**Comment by drewbitt** [comment]
Don't think it will ever go open source. That said, having at least the base daily goal setting - without any of the advance features like reviews, timeline etc - would be a welcome open source contribution without giving too much of their business model away.

**Comment by malcolmocean** [comment]
Yeah, it's not open source for various reasons, among them being that I don't actually want the responsibility of managing it as an open source project, and if I were to make it OS but not manage it, there's a good chance that some fork would runaway as the new canonical version. Also business reasons.

Regarding open-sourcing part of it, I agree that that would have fewer of the business model concerns, though it still runs into open source project management problems, and it also introduces a ton of new complexity, where now there are two versions or two parts of the codebase to maintain.

I have a vehement hatred for companies getting acquired and then the product getting ruined or scrapped. Hmm. Maybe I should have an explicit contract somewhere to represent that... where I give someone access to the source code and give them express permission to open source it if the app ever becomes unavailable downstream of acquisition (or me dying or anything else). (And I suppose, to have them agree to take responsibility for doing so!) And then anyone acquiring Complice would be told that this is the case and if they don't like it they can fuck off. And I would want them to fuck off—I wouldn't want to sell to someone who doesn't want to keep the app alive whether or not they continue to take responsibility for it, and this would be a good way to ensure that.

In fact, having now thought of this, I'm going to set this in motion at least to a degree that I'd have to _deal_ with it if Complice got acquired, by sending this issues ticket to a collaborator who already has access to the source code and indicating the above plan. I don't know if something more formal would have to be in place to make it not illegal if they were to do this open-sourcing after Complice gets acquired, but I've been wanting a solution for this for awhile and this feels like a step in the right direction and not something I'm likely to overlook if Complice were to get in serious acquisition talks.

**Comment by malcolmocean** [comment]
> I've been wanting a solution for this for awhile and this feels like a step in the right direction and not something I'm likely to overlook if Complice were to get in serious acquisition talks.

What I mean is that I trust myself to remember this thread and this plan if I were to start talking with people about acquiring Complice.
